app/controllers/api/v1/auth/passwords_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Api
+  module V1
+    module Auth
+      class PasswordsController < DeviseTokenAuth::PasswordsController
+        include ExceptionHandler
+        include ActAsApiRequest
+        include Api::V1::Concerns::AuthenticateUser
+
+        before_action :validate_redirect_url_param, only: []
+        before_action :validate_email_param, only: [:create]
```

Comment:
1. I have run the code analyzer before and now and it's not throwing any warnings on this. 
2 . It is necessary because i'm overriding a configuration inherited from DeviseTokenAuth::PasswordsController

Reply:
makes sense